packages/ui/src/ValueView/index.tsx

```diff
@@ -33,6 +35,18 @@ export interface ValueViewComponentProps<SelectedContext extends Context> {
    * numeraire.
    */
   priority?: PillProps['priority'];
+  /**
+   * If true, the asset icon will be hidden.
+   */
+  hideIcon?: boolean;
+  /**
+   * If true, the asset symbol will be hidden.
+   */
+  hideSymbol?: boolean;
+  /**
+   * If true, the displayed amount will be shortened.
+   */
+  abbreviate?: boolean;
```

Comment:
I'm not entirely sure if these kinds of props are in the spirit of the v2 UI component prop style

Reply:
Also these design changes should be validated on the storybook preview in the PR

Reply:
comment: validated storybook props in preview